contracts/libraries/JBErrors.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
```

Comment:
This file should only be for errors that are used across multiple contracts (index ob, unauthorized, etc.). All errors contained to a single contract should be declared in the same file.